src/dali/plugin/input/rest/coinbase.py

```diff
@@ -561,8 +561,8 @@ def _process_transfer(
             transaction_network = transaction[_NETWORK]
             crypto_hash: str = transaction_network[_HASH] if _HASH in transaction_network else Keyword.UNKNOWN.value
             if amount < ZERO:
-                if (
-                    _TO in transaction
+                if ( # pylint: disable=too-many-boolean-expressions
```

Comment:
Like this?  Not sure the rules for where this goes, but it seems like it is on the first line of an if.